docs/how-to-guides/create-backup.md

```diff
@@ -117,29 +121,27 @@ To restore data from a backup:
 
 ### 1. Lock the data directory
 
-To disable changes in the data directory, run the following query in `mgconsole` or Memgraph
-Lab:
+To disable file deletions in the data directory, run the following query in
+`mgconsole` or Memgraph Lab:
 
 ```cypher
 LOCK DATA DIRECTORY;
 ```
 
 ### 2. Copy the snapshot into the data directory
 
-Copy the data directory or, empty the directory, then copy a single WAL or
+Copy the snapshot or WAL directory or, empty it and then copy a single WAL or
 snapshot file into it.
 
 If you need help copying the files from the Docker container, check out the
 [Working with docker
-guide](/how-to-guides/work-with-docker.md##how-to-copy-files-from-and-to-a-docker-container).
+guide](/how-to-guides/work-with-docker.md#how-to-copy-files-from-and-to-a-docker-container).
 
-### 3. Unlock the directory
-
-Run the following query in `mgconsole` or Memgraph Lab to unlock the
-directory:
+### 3. Restart the instance
 
-```cypher
-UNLOCK DATA DIRECTORY;
-```
+By restarting the instance, Memgraph should restore the data from the files in
+the data directory. 
 
-Memgraph should restore the data from the files in the data directory. 
\ No newline at end of file
+Be sure to restart the instance before Memgraph automatically creates a new
+periodic snapshot because upon restart it will use that newer snapshot which is
+not the data you want to load. 
```

Comment:
```suggestion
periodic snapshot because upon restart it might use that newer snapshot which is
not the data you want to load. 
```

It depends on the lexicographically order of them. By default the files are named by the datetime they are created (something like `20230420132211347350_timestamp_1`), so if the users use the default names (really just copying the files without renaming), then what you wrote is totally true.